Streetcode/Streetcode.WebApi/Utils/SoftDeletingUtils.cs

```diff
@@ -20,12 +22,14 @@ public async Task DeleteStreetcodeWithStageDeleted(int daysLife)
             include: s => s.Include(x => x.Observers)
                            .Include(x => x.Targets));
 
-        if (streetcodes is null || streetcodes.Count() == 0)
+        var streetcodeContents = streetcodes as StreetcodeContent[] ?? streetcodes.ToArray();
```

Comment:
`_repositoryWrapper.StreetcodeRepository.GetAllAsync` returns non nullable `IEnumerable<T>` so you can just call `streetcodes.ToArray()` but in this case you only need to enumerate the collection so you don't need this logic at all

Reply:
Yes, we can immediately materialize a request for deferred methods without creating an additional variable. I was guided by the documentation then and missed this point a little. Thanks for the note, I'll fix it now.